pandas/plotting/_core.py

```diff
@@ -870,6 +870,8 @@ def _make_plot(self):
         scatter = ax.scatter(data[x].values, data[y].values, c=c_values,
                              label=label, cmap=cmap, **self.kwds)
         if cb:
+            if PATCH_MODE:
```

Comment:
I assume this ``PATCH_MODE` is just for debugging?

Reply:
yes, it should be removed. Didn't know of a better way to do this.